packages/core/tsup.config.ts

```diff
@@ -0,0 +1,40 @@
+import { defineConfig } from "tsup";
+import type { Plugin } from "esbuild";
+import { readFile } from "node:fs/promises";
+
+// Note: this is for putout because esbuild can't properly treeshake the
+// code, and is not aware that we do not use those dependencies.
```

Comment:
Shouldn't this comment be above `external:` (line 30)?

Reply:
Ha yes indeed :sweat_smile:
Here it should say

> There is no equivalent to require.resolve in esm, so we simplify this so that esbuild can do its magic